sympy/stats/rv_interface.py

```diff
@@ -216,7 +216,9 @@ def kurtosis(X, condition=None, **kwargs):
     3 means that the distribution produces fewer and less extreme outliers
     than the normal distribution.
 
-    kurtosis(X) = E( ((X - E(X))/sigma)**4 )
+    kurtosis(X) = E(((X - E(X))/sigma)**4)
+
+    .. [1] https://en.wikipedia.org/wiki/Kurtosis
```

Comment:
Add this to a section References as in my example. (Just paste it at the end of the documentation string.)